tests/unit/mixed/async_compat/test_network.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) "Neo4j"
+# Neo4j Sweden AB [https://neo4j.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import annotations
+
+import asyncio
+import socket
+import typing as t
+
+import freezegun
+import pytest
+
+from neo4j._async_compat.network import AsyncBoltSocket
+from neo4j._exceptions import SocketDeadlineExceededError
+
+from ...._async_compat.mark_decorator import mark_async_test
+
+
+if t.TYPE_CHECKING:
+    import typing_extensions as te
+    from freezegun.api import (
+        FrozenDateTimeFactory,
+        StepTickTimeFactory,
+        TickingDateTimeFactory,
+    )
+
+    TFreezeTime: te.TypeAlias = (
+        StepTickTimeFactory | TickingDateTimeFactory | FrozenDateTimeFactory
+    )
+
+
+@pytest.fixture
+def reader_factory(mocker):
+    # class MockReader(mocker.Mock):
```

Comment:
What's the purpose of the commented out lines in this file? Is there a good chance they'll be needed later or are they just a remnant of some template?

Reply:
The purpose is to test that a reviewer pays attention... 

<details>
<summary>Alternative explanation</summary>

Or *maaaaybe* I just forgot cleaning up properly :sweat_smile: 

</details>